src/styles/form.scss

```diff
@@ -106,19 +146,15 @@
 }
 
 .submitButton {
-  background-color: #f3f0f0;
-  border: 1px solid #969696;
-  border-radius: 7px;
-  color: black;
+  background-color: #4594fd;
+  border-radius: 2px;
+  color: #FFFFFF;
```

Comment:
change to #ffffff - we have a linting rule for lower-case hex values

Reply:
Ah ok, will do